src/App.js

```diff
@@ -137,8 +139,8 @@ class AlbumCover extends Component {
   }
 
   render() {
-    const src = this.props.track.album.images[0].url;
-    return (<img src={src} style={{ width: 400, height: 400 }} />);
+    const coverImageSrc =  this.props.track.album.images.length > 0 ? this.props.track.album.images[0].url : noCoverImage;
```

Comment:
What if track is empty and does not contain an album? I see nothing in the code proving that it cannot happen.